### ISSUE TITLE
Would like to be able to turn off "Navigation finished" notification

### ISSUE THREAD

**Issue by martenjj** [osmandapp/Osmand#6341]
The "Navigation has finished" notification popup displayed when the destination has been reached has some options that may be useful for some, but personally I never use them.  The popup, and having to close it manually every time, is annoying.  Most other car and handheld navigation systems seem to simply exit navigation mode (sometimes after a short delay) when the destination is reached.

Suggestions:  (a) Provide an option to not show this popup at all.  There is a hint that an option exists to turn it off (shown in the the screen shot at issue #6225), but I cannot find this option anywhere within the app or navigation settings.

or (b) Automatically close the popup after a delay (e.g. 10 seconds).

App version: 3.2.7

**Comment by pebogufi** [comment]
- the screenshot at issue #6225 shows a situation when a quick action button is created with "start/stop nav". 
- this option does not work. If it is set to off, the message is also shown, at least in simulation mode.

**Comment by matoromoto** [comment]
For me personally (and i'm sure for everybody using osmand on a motorcycle) this is more than a nice to have. 
It' easy to wipe away the 'navigation finished' card while driving in a car. But it is impossible to do so on a motorcycle. You need to stop (not possible everywhere), take off your gloves, wipe away the card, get your gloves on again, start riding.

**Comment by scaidermern** [comment]
I fully agree. On a motorcycle this notification is really annoying and can lead to dangerous actions while driving. Consider closing it automatically after a few seconds to return back to the map.

**Comment by vshcherb** [comment]
Liked @scaidermern idea, I think about 20-30 seconds
